src/Shadow.ts

```diff
@@ -19,12 +20,12 @@ export const shadowDefaultValues: Partial<TClassProperties<Shadow>> = {
 };
 
 type TShadowSerializedProps = {
-  color?: string;
-  blur?: number;
-  offsetX?: number;
-  offsetY?: number;
-  affectStroke?: boolean;
-  nonScaling?: boolean;
+  color: string;
+  blur: number;
+  offsetX: number;
+  offsetY: number;
+  affectStroke: boolean;
+  nonScaling: boolean;
```

Comment:
Those needs to be optional because toObject needs a return type, and it doesn't return the full object every time

Reply:
All serialized props are defined as non optional and used with Partial
So I made this the same

Reply:
And in toObject is does return a partial in case of `includeDefaultValues===false`